jvm/ml4ir-inference/scripts/generate_model.sh

```diff
@@ -20,9 +20,19 @@ $EXECUTOR ${CLASSIFICATION_MODEL}/pipeline.py \
     --data_dir ${CLASSIFICATION_MODEL}/tests/data/csv/  \
     --feature_config ${CLASSIFICATION_MODEL}/tests/data/configs/feature_config.yaml \
     --model_config ${CLASSIFICATION_MODEL}/tests/data/configs/model_config.yaml \
-    --run_id ${1} \
+    --run_id ${1}_classification \
     --data_format csv \
     --execution_mode train_inference_evaluate \
     --batch_size 32
 
+
+RANKING_MODEL=ml4ir/applications/ranking
+
+$EXECUTOR ${RANKING_MODEL}/pipeline.py \
+    --data_dir ${RANKING_MODEL}/tests/data/csv \
+    --feature_config ${RANKING_MODEL}/tests/data/configs/feature_config.yaml \
+    --run_id ${1}_ranking \
+    --data_format csv \
+    --execution_mode train_inference_evaluate \
```

Comment:
I recommend adding `--loss_key rank_one_listnet` here as well. This is the most commonly used loss internally.

Reply:
Ok, I'll add this.